todocore/src/main/java/com/knighttodo/todocore/rest/DayTodoResource.java

```diff
@@ -175,4 +178,26 @@ public DayTodoReadyResponseDto updateIsReady(@PathVariable UUID dayId, @PathVari
             throw new DayTodoReadyCanNotBeUpdatedException("Day todo ready can't be updated.", ex);
         }
     }
+
+    @PatchMapping
+    @ResponseStatus(HttpStatus.OK)
+    @ApiOperation(value = "Change orderNumber id existed Todo", response = DayTodoResponseDto.class, responseContainer = "List")
+    @ApiResponses(value = {
+            @ApiResponse(code = 201, message = "Created"),
+            @ApiResponse(code = 400, message = "Invalid operation"),
+            @ApiResponse(code = 403, message = "Operation forbidden"),
+            @ApiResponse(code = 500, message = "Unexpected error")
+    })
+    public List<DayTodoResponseDto> changeDayTodoOrderNumber(@PathVariable Map<UUID,Integer> orderNumbersMap) {
+        try {
+            orderNumbersMap.keySet().stream().map(dayTodoService::findById);
+            List<DayTodoVO> dayTodoVOS = orderNumbersMap.entrySet().stream()
+                    .map(entry ->dayTodoService.orderNumberUpdate(entry.getKey(),entry.getValue()))
+                    .collect(Collectors.toList());
+            return dayTodoVOS.stream().map(dayTodoRestMapper::toDayTodoResponseDto).collect(Collectors.toList());
```

Comment:
This should be done in Service. In Resource you should accept request, validate it with annotations in method params or in Dtos and send all information to service. d